src/logic/calculate.test.js

```diff
@@ -0,0 +1,39 @@
+import calculate from "./calculate";
```

Comment:
- [ ] Based on the requirements, all functionalities and scenarios of the calculate.js module should be covered using jest. Let's kindly make sure the following scenarios are equally tested:
- (buttonName === '.') 
- (buttonName === '+/-')